README.md

```diff
@@ -1,50 +1,120 @@
-OKR report aggregation tool
-=======
+<h1 align="center">
+  okra
+</h1>
 
-Prototype aggregation tool for markdown-based OKR reports. This is work in progress.
+<p align="center">
+  <strong>Aggregation tool for markdown-based activity reports.</strong>
+</p>
 
-## Running it
+<p align="center">
+  <!--
+  <a href="https://ocaml.ci.dev/github/tarides/okra">
+    <img src="https://img.shields.io/endpoint?url=https://ocaml.ci.dev/badge/tarides/okra/main&logo=ocaml" alt="OCaml-CI Build Status" />
+  </a>
+  -->
 
-The code reads data from stdin, aggregates per KR and outputs the result.
+  <a href="https://github.com/tarides/okra/actions/workflows/build.yml">
+    <img src="https://github.com/tarides/okra/actions/workflows/build.yml/badge.svg?branch=main" alt="Build Status" />
+  </a>
+</p>
 
-Example:
+`okra` can:
+
+ - Generate a weekly report stub with your Github activity
+ - Lint reports to check they are in the expected format
+ - Aggregate weekly reports for multiple engineers and projects
+
+## Getting Started
+
+### Installation
+
+Install opam if you don't already have it, and add [`tarides/opam-repository`](https://github.com/tarides/opam-repository) to your list of opam repositories:
+
+Either only to the current opam switch with the command:
+```sh
+opam repository add tarides https://github.com/tarides/opam-repository.git
 
-```
-# cat okr-report1.md okr-report2.md | okra | less
 ```
 
-If there are warnings that are not expected (i.e not about the "OKR updates" section), they should be manually resolved. There's also a script that handles some common formatting issues by fixing indents, bulletpoints etc (fmt.sh). To use it:
+Or to the list of opam repositories for all opam switches with the command:
+```sh
+opam repository add --all tarides https://github.com/tarides/opam-repository.git
+```
 
+However, note that if you use `-all` as above, you will then have to
+instruct opam to also search in that repository in each switch, as the list
+of repositories opam is considering is switch-specific. Once you are in a
+given switch, this can be done with:
+```sh
+opam remote set-repos default tarides
+```sh
+where the repositories are listed in search order, i.e. here we ask opam to
+look in its `default` repository first and in the `tarides` repository after
+that.
```

Comment:
```suggestion
However, note that if you use `--all` as above, you will then have to
instruct opam to also search in that repository in each switch, as the list
of repositories opam is considering is switch-specific. Once you are in a
given switch, this can be done with:
```sh
opam remote set-repos default tarides
```
where the repositories are listed in search order, i.e. here we ask opam to
look in its `default` repository first and in the `tarides` repository after
that.
```

Reply:
It's not clear in the GitHub UI, but I'm suggesting using double dashes for the `--all` option, and the terminating maker for the sh markdown code block shouldn't indicate the language (remove `sh`).